charts/vault/templates/post-pod-creation-cm.yaml

```diff
@@ -8,9 +8,9 @@ data:
       # configuring kubernetes auth method and use it later to create roles.
       vault auth enable kubernetes
 {{- if semverCompare "<1.21-0" .Capabilities.KubeVersion.Version }}
-      vault write auth/kubernetes/config kubernetes_host="https://kubernetes.default.svc:443"
+      vault write auth/kubernetes/config kubernetes_host="https://kubernetes.default.svc:443" disable_iss_validation=true
```

Comment:
why do you set it for Kubernetes versions less than 1.21? The Valut's recommendation is for "Kubernetes v1.21+"